controllers/drcluster_controller_test.go

```diff
@@ -186,6 +186,42 @@ func inspectClusterManifestSubscriptionCSV(match bool, value string, drcluster *
 	}
 }
 
+func updateDRClusterParameters(clusterName string, clusterFence ramen.ClusterFenceState,
+	cidrs []string, s3ProfileName string,
+) *ramen.DRCluster {
+	drclusterLookupKey := types.NamespacedName{
+		Name: clusterName,
+	}
+	latestdrc := &ramen.DRCluster{}
+
+	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		var err error
+
+		err = apiReader.Get(context.TODO(), drclusterLookupKey, latestdrc)
+		if err != nil {
+			return err
+		}
+
+		latestdrc.Spec.ClusterFence = clusterFence
```

Comment:
Why not check for nil clusterFence in the same way we check for empty s3ProfileName and cidrs?

Reply:
It is not nil, but empty string. And empty string is a valid value